blaze/server/server.py

```diff
@@ -1,6 +1,10 @@
 from __future__ import absolute_import, division, print_function
 
 import socket
+try:
+    import cPickle as pickle
+except ImportError:
+    import pickle
```

Comment:
I know, I've just found myself doing this exact same try-except block.  Just a thought that we could do this in `compatibility.py` and then import pickle from there.  Fine either way though.